third_party/terraform/utils/provider.go.erb

```diff
@@ -156,6 +156,7 @@ func Provider() terraform.ResourceProvider {
 				"google_compute_route":                         resourceComputeRoute(),
 				"google_compute_router":                        resourceComputeRouter(),
 				"google_compute_router_interface":              resourceComputeRouterInterface(),
+				"google_compute_router_nat":                    resourceComputeRouterNat(),
```

Comment:
can you also add beta blocks around this one?

Reply:
Done.